neurom/stats.py

```diff
@@ -176,3 +176,26 @@ def compare_two(data1, data2, test=StatTests.ks):
     Stats = namedtuple('Stats', ['dist', 'pvalue'])
 
     return Stats(*results)
+
+
+def total_score(paired_dats, p=2, test=StatTests.ks):
```

Comment:
True! We could add a test to check if p is correctly interpreted. (Example: results for p=inf, p=1 should be different when distributions are not the same).